### PR TITLE
Migrating 2122 Form Changes from Feature Branch into Master

### PR DESCRIPTION
**Pull request by bastosmichael**
## Description of change
Adding additional sections to the 2122 form with signature blocks for PDF generation.

## Original issue(s)
https://vajira.max.gov/browse/API-1294

### CODE REVIEW

File: modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    def upload_to_vbms(power_of_attorney, path)
+      uploader = VbmsUploader.new(
+        filepath: path,
+        file_number: power_of_attorney.auth_headers['va_eauth_pnid'],
+        doc_type: '295'
+      )
+      uploader.upload!
+    rescue VBMS::Unknown
+      rescue_vbms_error(power_of_attorney)
```
Comment (edmangimelli) [inline modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb:32]: this method is missing
(in console)
```
> ClaimsApi::PoaFormBuilderJob.new.methods.include? :rescue_vbms_error
=> false
```

File: modules/claims_api/lib/claims_api/vbms_uploader.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        file_number: @file_number
+      )
+
+      upload_response = upload_document(
+        filepath: @filepath,
+        upload_token: upload_token_response.upload_token
+      )
+      {
+        vbms_new_document_version_ref_id: upload_response.upload_document_response[:@new_document_version_ref_id],
+        vbms_document_series_ref_id: upload_response.upload_document_response[:@document_series_ref_id]
```
Comment (edmangimelli) [inline modules/claims_api/lib/claims_api/vbms_uploader.rb:23]: the response returns keys that start with `@` ? super weird

Comment (charleystran) [reply modules/claims_api/lib/claims_api/vbms_uploader.rb:23]: yes, thats what comes back from VBMS. Very weird but thats the key

Comment (edmangimelli) [reply modules/claims_api/lib/claims_api/vbms_uploader.rb:23]: stop working, Charley

**Comment by edmangimelli** [comment]
it might be nice to address the 5 codeclimate issues, but I'm fine with rolling that into some tech-debt tickets if we want

File: modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb:32]: Thanks for catching, might have been a merge issue from another PR, I'll address it.

File: modules/claims_api/lib/claims_api/vbms_uploader.rb (hunk 2 above)
Comment (bastosmichael) [reply modules/claims_api/lib/claims_api/vbms_uploader.rb:23]: lol

File: modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb]: Refactored the whole thing into it's own module and so it should see the method now...

Comment (edmangimelli) [reply modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb:24]: nice!

**Pull request merged**